change/@fluentui-react-examples-1b3a1730-975a-4600-bb36-969e82da5492.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update snapshots to reflect beak hc styling",
+  "packageName": "@fluentui/react-examples",
+  "email": "ololubek@microsoft.com",
+  "dependentChangeType": "patch"
```

Not very important but this should be none since it's only test changes
```suggestion
  "type": "none",
  "comment": "Update snapshots to reflect beak hc styling",
  "packageName": "@fluentui/react-examples",
  "email": "ololubek@microsoft.com",
  "dependentChangeType": "none"
```

---

Aaaand now I know to check whether automerge is enabled before approving with comments, oops. 🤦‍♀️  At least this one wasn't very important.